proto/opamp.proto

```diff
@@ -817,3 +833,33 @@ message AgentConfigFile {
     // example "text/yaml".
     string content_type = 2;
 }
+
+/////////////////////////////////////////////////////////////////////////////////////
+// Custom messages
+/////////////////////////////////////////////////////////////////////////////////////
+
+message CustomCapabilities {
+    // A list of custom capabilities that are supported. Each capability is a reverse FQDN
+    // with optional version information that uniquely identifies the custom capability
```

Comment:
>is a reverse FQDN with optional version information

This part seems underspecified. My understanding is that FQDN usage is what ensure that are no clashes between custom messages introduced by different agents. For this to work correctly we need to define which part of the string is the FQDN and which part is the message. I can see that we have an example that clarifies it, but I do not think just an example is sufficient. I think we also need some wording here that specifies the format of this string.

Reply:
Let me know if you think the new language resolves this.